spt/OrangeBox/overlay/overlay-renderer.cpp

```diff
@@ -0,0 +1,122 @@
+#include "stdafx.h"
+
+#ifdef SSDK2007
+
+#include "overlay-renderer.hpp"
+#include "ivrenderview.h"
+#include "..\..\game\client\iviewrender.h"
+#include "view_shared.h"
+#include "..\modules\ClientDLL.hpp"
+#include "..\modules\EngineDLL.hpp"
+#include "..\modules.hpp"
+#include <SDK\hl_movedata.h>
+#include "..\cvars.hpp"
+#include "overlays.hpp"
+
+OverlayRenderer g_OverlayRenderer;
+const float ASPECT_RATIO = 16.0f / 9.0f;
+const int VIEW_CLEAR = 1;
+const int VIEWMODEL_MASK = ~RENDERVIEW_DRAWVIEWMODEL;
+
+bool OverlayRenderer::shouldRenderOverlay()
+{
+	return _y_spt_overlay.GetBool() && !engineDLL.Demo_IsPlayingBack();
+}
+
+bool OverlayRenderer::shouldFlipScreens()
+{
+	return _y_spt_overlay_swap.GetBool();
+}
+
+void OverlayRenderer::modifyBigScreenFlags(int & clearFlags, int & drawFlags)
```

Comment:
Please fix the whitespace in the `type* name` and `type& name` here and elsewhere in this file.